### DISCUSSION TITLE
SLSKD and SMB shares?

### DISCUSSION THREAD

**Discussion by earwigcahootz** [mrusse/Slsk-Upload-Stats-Tracker#15] [Q&A] [answered]
Can this be used with SLSKD logs?
If the shared folders are located on a separate device and presented as SMB shares on the network, can the installation be on a separate networked PC with access to the SMBs?

**Comment by mrusse** [comment, marked as answer]
Unfortunately SLSKD logs are not supported. It can however access SMB shares for the shared folders.
